perun-core/src/main/java/cz/metacentrum/perun/core/entry/ServicesManagerEntry.java

```diff
@@ -298,6 +298,20 @@ public void deleteService(PerunSession sess, Service service) throws ServiceNotE
 		getServicesManagerBl().deleteService(sess, service);
 	}
 
+	@Override
+	public void deleteService(PerunSession sess, Service service, boolean forceFlag) throws ServiceNotExistsException, PrivilegeException, RelationExistsException, ServiceAlreadyRemovedException {
+		Utils.checkPerunSession(sess);
+
+		// Authorization
+		if (!AuthzResolver.authorizedInternal(sess, "deleteService_Service_policy", service)) {
```

Comment:
We usually create a new policy for new methods.
```suggestion
		if (!AuthzResolver.authorizedInternal(sess, "deleteService_Service_boolean_policy", service)) {
```

Reply:
Done, although by renaming the old policy, because according to the comments below, I removed the original method altogether.